src/gui/newwizard/setupwizardcontroller.h

```diff
@@ -36,7 +37,7 @@ class SetupWizardController : public QObject
     Q_OBJECT
 
 public:
-    explicit SetupWizardController(QWidget *parent);
+    explicit SetupWizardController(SettingsDialog *parent);
```

Comment:
These changes appear unrelated to the original PR purpose.

Reply:
Hm yes, so it won't hurt